client/main.lua

```diff
@@ -23,24 +27,35 @@ end)
 
 Citizen.CreateThread(function()
     while true do
-        Citizen.Wait(0)
-
-        for k in pairs(Config.ProximityAlarms) do
-
-        local playerCoords = GetEntityCoords(GetPlayerPed(-1), false)
-        local dist = Vdist(playerCoords.x, playerCoords.y, playerCoords.z, Config.ProximityAlarms[k].x, Config.ProximityAlarms[k].y, Config.ProximityAlarms[k].z)
-
-        if dist <= Config.Distance then
-			local playerPed = GetPlayerPed(-1)
-			if intruderDetected == false and PlayerData.job.name ~= Config.Job then
-				local x,y,z = table.unpack(GetEntityCoords(GetPlayerPed(-1), false))
-				TriggerServerEvent('esx_phone:send', Config.Job, 'An intruder is within ' ..Config.Distance.. 'm of your hideout ', true, {x =x, y =y, z =z})
-			end															
-		
-			intruderDetected = true
-			Citizen.Wait(Config.AlertTime)
-			intruderDetected = false
+		Citizen.Wait(0)
+		local playerPed = GetPlayerPed(-1)
+		local playerCoords = GetEntityCoords(playerPed, false)
+		local dist
+
+		for k,v in pairs(Config.ProximityAlarms) do
+			for k2,v2 in pairs(v) do 
+				dist = Vdist(playerCoords.x, playerCoords.y, playerCoords.z, v.coords)
+
+				if dist <= v.alertDistance then
+					isInZone = true
+					if isInZone == true and PlayerData.job.name ~= v.whitelistedJob then
+						TriggerServerEvent('esx_motiondetector:Broadcast', playerCoords)
+					end															
+				
+					Citizen.Wait(Config.AlertTime)
+					isInZone = false
+				end
 			end
-        end
+		end
     end
 end)
+
+RegisterNetEvent('esx_motiondetector:Notify')
+AddEventHandler('esx_motiondetector:Notify', function(coords)
+	if Config.NotificationType == 1 then
+		TriggerServerEvent('esx_phone:send', Config.Job, 'An intruder is within ' ..Config.Distance.. 'm of your hideout ', true, {x = coords.x, y = coords.y, z = coords.z})
+	elseif Config.NotificationType == 2 then
+		ESX.ShowNotification('An intruder is within ' ..Config.Distance.. 'm of your hideout ', false, true)
```

Comment:
`Config.Distance` will always result in an error since you deleted `Config.Distance` in the `config.lua`

Reply:
Yeah just from the notifications though. I changed up the notification to resolve this. Thanks!